migrations/3_deploy_contracts.js

```diff
@@ -21,7 +21,8 @@ module.exports = function(deployer, network) {
         await lpDeployer.deployAndRegister(Minter, "Minter", controller.address, config.minter.inflation, config.minter.inflationChange, config.minter.targetBondingRate)
         await lpDeployer.deployAndRegister(LivepeerVerifier, "Verifier", controller.address, config.verifier.solvers, config.verifier.verificationCodeHash)
 
-        if (!lpDeployer.isMainNet(network)) {
+        if (!lpDeployer.isProduction(network)) {
```

Comment:
If we are deploying to main net or doing a dry run on a test network we will not deploy a faucet